backend/src/Edna/Config/Definition.hs

```diff
@@ -62,17 +64,24 @@ data LoggingConfig =
   -- ^ No logging.
   deriving stock (Generic, Show, Eq)
 
+-- | Parse LoggingConfig
+parseLoggingConfig :: e -> String -> Either e LoggingConfig
+parseLoggingConfig e p = case T.toLower (T.pack p) of
+  "dev"     -> Right LogDev
+  "prod"    -> Right LogProd
+  "nothing" -> Right LogNothing
+  _ -> Left e
+
 loggingConfigAesonOptions :: Aeson.Options
 loggingConfigAesonOptions = Aeson.defaultOptions
   { Aeson.sumEncoding = Aeson.UntaggedValue
-  , Aeson.constructorTagModifier = drop 3
   }
 
-instance Aeson.FromJSON LoggingConfig where
-  parseJSON = Aeson.genericParseJSON loggingConfigAesonOptions
+instance Aeson.FromJSON LoggingConfig
 
 instance Aeson.ToJSON LoggingConfig where
   toEncoding = Aeson.genericToEncoding loggingConfigAesonOptions
+  toJSON = Aeson.toJSON . T.toLower . T.drop 3 . show
```

Comment:
Erm, it looks wrong.
1. Why did you remove `Aeson.constructorTagModifier` from `loggingConfigAesonOptions`? It used to drop `Log` prefix, but now it doesn't.
2. `toEncoding` and `toJSON` should have identical definitions, but now the are different. `toJSON` drops the prefix, but `toEncoding` doesn't. You can define `toJSON = Aeson.genericToJSON loggingConfigAesonOptions` to make it identical.
3. ` instance Aeson.FromJSON LoggingConfig` with no explicit definition will likely use default parsing options which I guess do not drop prefix.  Since we have `parseLoggingConfig` now, I think it's better to define `parseJSON` in terms of `parseLoggingConfig`.

Reply:
Fixed.
I'll read a bit more about Aeson on weekends)